plotter/heat.go

```diff
@@ -57,6 +58,10 @@ type HeatMap struct {
 	// Min and Max define the dynamic range of the
 	// heat map.
 	Min, Max float64
+
+	// Rasterized is a flag that indicates whether the heatmap
+	// should be produced using raster-based drawing or not
```

Comment:
```suggestion
	// Rasterized indicates whether the heatmap
	// should be produced using raster-based drawing or not.
```

Reply:
As @sbinet suggests, but delete the trailing " or not".